third_party/conan/configs/windows/profiles/msvc2017_debug_x86

```diff
@@ -10,6 +10,7 @@ build_type=Debug
 [options]
 OrbitProfiler:system_qt=False
 OrbitProfiler:with_gui=False
+OrbitProfiler:with_orbitgl=False
```

Comment:
I was wondering where are these profiles used? This is 32 bit windows is it?

Reply:
Yeah. We once had this discussion that we wanted to have a 32bit build for preserving OrbitDll's functionality (which needs 32bit support).

But since we removed OrbitDll for now anyway, I also feel that we should remove those. But I didn't want to do it in this PR.